api-gateway/src/main/resources/application-prod.yml

```
@@ -0,0 +1,23 @@
+      on-profile: prod
+
+  cloud:
+    gateway:
+      httpclient:
+        connect-timeout: 2000 # 빠른 연결 실패 유도
+        response-timeout: 3s
+      globalcors:
+        cors-configurations:
+          '[/**]':
+            allowedOrigins: "https://tmp-domain.com"
+            allowedHeaders: [ "Content-Type", "Authorization" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)